README.md

```diff
@@ -1 +1,29 @@
-# oop-school-library
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# School Library
+
+> An Educational project with mock functionality of a school library.
+
+## Built With
+
+- Ruby
+
```

Comment:
- [ ] Kindly add the `Getting Started` section and add the instruction on how to clone your repository, this will make your repository to be more informative.